techniques/about.html

```diff
@@ -85,52 +85,52 @@ <h2>Change Log</h2>
 
                         <li>
 
-                            <time datetime="2019-09-12">Sept 12th 2019</time>: Added <a href="./techniques/failures/F105">Failure of Success Criterion 2.5.1 due to providing functionality via a path-based
+                            <time datetime="2019-09-12">Sept 12th 2019</time>: Added <a href="./Techniques/failures/F105">Failure of Success Criterion 2.5.1 due to providing functionality via a path-based
```

Comment:
I think these still look off - this containing doc is already within the `/Techniques/` path

Reply:
ah, good catch @dbjorge - indeed, I just did a find/replace for the uppercase `T`, but it seems these are broken even further even in the live version at the moment https://www.w3.org/WAI/WCAG22/Techniques/about

will push an update (for the about document only, which i think is the only one that's still broken this way)

Reply:
actually...wondering if we want to kill of this (non-maintained) list of "new and updated techniques" and just reference github's change log instead

Reply:
doh, this smelled familiar ... already did this exact change (removing the non-maintained list) over here https://github.com/w3c/wcag/pull/3588

Reply:
re-applying #3588 here (i hadn't merged in the changes we did there before, done now), this circumvents/closes this problem outright